packages/react/src/utils/useAnchorPositioning.ts

```diff
@@ -272,14 +272,17 @@ export function useAnchorPositioning(
 
   const renderedSide = getSide(renderedPlacement);
   const renderedAlignment = getAlignment(renderedPlacement) || 'center';
-  const hidden = Boolean(hideWhenDetached && middlewareData.hide?.referenceHidden);
+  const anchorHidden = Boolean(hideWhenDetached && middlewareData.hide?.referenceHidden);
 
   const positionerStyles = React.useMemo(
     () => ({
       ...floatingStyles,
-      ...(hidden && { visibility: 'hidden' as const }),
+      ...(anchorHidden && {
+        visibility: 'hidden' as const,
+        pointerEvents: 'none' as const,
+      }),
```

Comment:
`pointerEvents` was being set inside each positioner, but since `visibility` was inlined as a functional style, it can be done here too. We don't expose a style hook for this state but could later, but I don't think it needs to be customized.